### DISCUSSION TITLE
视频内显示时间是preview ONLY？

### DISCUSSION THREAD

**Discussion by yuijker** [the1812/Bilibili-Evolved#3447] [提问 / Q&A] [answered]
今天我在在线仓库里看到视频内显示时间这个组件，但这个组件在master分支下下载是一直出错，显示以下内容：

![image](https://user-images.githubusercontent.com/72558803/174961614-c5602b1d-1e7b-41e4-83d2-28ca11169cae.png)

注意右下角。
但是将分支换成preview后就没有问题了。这是怎么了？

**Comment by the1812** [comment]
似乎是 @FoundTheWOUT 没把组件更新到他的 master 分支上

**Comment by FoundTheWOUT** [comment, marked as answer]
已经更新到主分支了👌
